deploy/database/drop_databases.sql

```diff
@@ -1,2 +0,0 @@
-DROP DATABASE IF EXISTS buttonmen;
-DROP DATABASE IF EXISTS buttonmen_test;
```

Comment:
If you look at the function recreate_database() in database_initialise.php, you'll see that the drop actually happens there. Thus, we don't need drop_databases.sql any more for the AMP stacks.

You also don't seem to use it in your initialise_schema script, so it's orphaned, as far as I can tell.